source/renderer/app/components/staking/stake-pools/FilterPopOver.js

```diff
@@ -0,0 +1,253 @@
+// @flow
+/* eslint-disable jsx-a11y/label-has-associated-control, jsx-a11y/label-has-for */
```

Comment:
@yakovkaravelov Please remove disabled eslint rules and apply changes to fix them